src/dotnet/commands/dotnet-list/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Tools.List.ProjectToProjectReferences;
+
+namespace Microsoft.DotNet.Tools.List
+{
+    public class ListCommand : DispatchCommand
+    {
+        protected override string HelpText => @".NET List Command
+
+Usage: dotnet list [options] <object> <command> [[--] <arg>...]]
+
+Options:
+  -h|--help  Show help information
+
+Arguments:
+  <object>   The object of the operation. If a project file is not specified, it defaults to the current directory.
```

Comment:
True. However, `dotnet list` can also work on a project **or** a solution file, right? It is the `<command>` that specializes it. So, we need to include both of these in the description. 

`<object>       The solution or the project file to list items from. If not specified, it defaults to finding the solution or the project file in the current directory.`